controllers/toplevel/index.js

```diff
@@ -20,20 +20,47 @@ const robots = require('../../config/app/robots.json');
 if (app.get('env') !== 'production') {
     robots.push('/');
 }
+const assets = require('../../modules/assets');
 
 const legacyUrl = config.get('legacyDomain');
 
 const newHomepage = (req, res) => {
     // don't cache this page!
     res.cacheControl = { maxAge: 0 };
 
-    let serveHomepage = news => {
-        let lang = req.i18n.__('toplevel.home');
+    const serveHomepage = news => {
+        const lang = req.i18n.__('toplevel.home');
+
+        const heroImageDefault = {
+            small: assets.getCachebustedPath('images/hero/home-hero-1--small.jpg'),
+            large: assets.getCachebustedPath('images/hero/home-hero-1--large.jpg'),
+            default: assets.getCachebustedPath('images/hero/home-hero-1--small.jpg'),
+            caption: 'Cloughmills Community Action, Grant £4,975*'
+        };
```

Comment:
Half-wondering if we should make this a function that will complain if we miss out all the parameters etc? It could also do the image cachebusting in one go, too, so you'd just need to pass the paths.

Reply:
... would also be easier to later extend it to accept a translated caption for the inevitable time we need do this.

Reply:
Great shout, I've done that now 💯